packages/hoc-react-loader-examples/package.json

```diff
@@ -11,6 +12,7 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "ci": "npm-run-all --parallel test"
```

Comment:
I wait for this [PR](https://github.com/facebookincubator/create-react-app/pull/2729) to be merged to add linting to `ci` script.